markovflow/gauss_markov.py

```diff
@@ -215,3 +215,31 @@ def check_compatible(dist_1: GaussMarkovDistribution, dist_2: GaussMarkovDistrib
     tf.debugging.assert_equal(dist_1.state_dim, dist_2.state_dim)
     tf.debugging.assert_equal(dist_1.batch_shape, dist_2.batch_shape)
     tf.debugging.assert_equal(dist_1.num_transitions, dist_2.num_transitions)
+
+
+class BTDGaussian:
```

Comment:
for consistency with the rest of the toolbox,
it might be best to name this
BlockTriDiagonalGaussian